docs/guide/replacement-guides/chalk.md

```diff
@@ -74,3 +74,26 @@ Similarly, you can use RGB and hex colors:
 console.log(chalk.rgb(239, 239, 239)('Hello world!')) // [!code --]
 console.log(ansis.rgb(239, 239, 239)('Hello world!')) // [!code ++]
 ```
+
+## Browser support
+
+While these libraries are primarily designed for terminal output, some projects may need colored output in browser environments.
+
+Following [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/API/console#styling_console_output), the native approach is `%c` directive in `console.log`:
+
+```js
+console.log(
+  'Hello %ce%c1%c8%ce',
+  'color: #ec8f5e;',
+  'color: #f2ca60;',
+  'color: #bece57;',
+  'color: #7bb560;',
+  'Ecosystem Performance'
+)
+```
+
+Library support:
```

Comment:
i'm not sure we need this list, especially listing chalk since this doc is reached by users looking to _drop_ chalk

maybe we just need to mention something like:

> If you need a package which works both in terminals and in browsers, consider using ansis.

since it seems no other package supports browsers?